lib/samples/densify_and_generalize_geometry/README.md

```diff
@@ -0,0 +1,35 @@
+# Densify and generalize geometry
+
+A multipart geometry can be densified by adding interpolated points at regular intervals. Generalizing multipart geometry simplifies it while preserving its general shape. Densifying a multipart geometry adds more vertices at regular intervals.
```

Comment:
Could it be shortened for everyone in the common-samples repository? The 3rd sentence is pretty redundant to the 1st sentence.

Reply:
There's still inconsistency in the length across the samples - Show grid is another, and there'll be more as we get through the 200+ of them. The original purpose was for the readme rather than the sample viewer UI. Plus It'd be quite a bit of work to go through every sample and apply some kind of length rule.

I'd lean towards leaving it for now. Swift display the name only and then the description if you tap on an `i` button and expand, so there's things we could do to tidy it up.

Reply:
Yeah the (i) button would be great. We can wait for that.